mesh_geometry_datagather_utils.py

```diff
@@ -158,25 +158,45 @@ def parse_obj_to_geometrydata(meshObj, parentSkeleton, shaderIndex, correctedNor
         uvlayer = bm.loops.layers.uv[0]
         uvlayer2 = bm.loops.layers.uv[1]
 
+    #vertex color layers (should be two but who knows what people are doing)
+    vcLayer = None
+    vcLayer2 = None
+
+    if len(bm.loops.layers.color) > 0:
+        vcLayer = bm.loops.layers.color[0]
+
+        if len(bm.loops.layers.color) > 1:
+            vcLayer2 = bm.loops.layers.color[1]
+
     #fill uvCoords and qtangents with blank entries so that we can fill them in any order
-    geom.uvCoords = [(0.0, 0.0)] * len(geom.vertPositions)
-    geom.uvCoords2 = [(0.0, 0.0)] * len(geom.vertPositions)
+    geom.uvCoords = [(1.0, -1.0)] * len(geom.vertPositions)
+    geom.uvCoords2 = [(1.0, -1.0)] * len(geom.vertPositions)
+    geom.vColor = [(0.0, 0.0, 0.0, 0.0)] * len(geom.vertPositions)
+    geom.vColor2 = [(0.0, 0.0, 0.0, 0.0)] * len(geom.vertPositions)
```

Comment:
Ah sorry.. that is just me being stupid.

Reply:
The actual issue is in export_mesh.py where I did just multiply the value 255 times :D.

Reply:
Cool! I just tested it and had to set vColor (1) to be all 1.0, because apparently the ped's shading gets really messed up if it's all zeroes. vColor2 being zero appears to be fine though